modules/opening-default-network-ports-udn.adoc

```diff
@@ -0,0 +1,31 @@
+//module included in the following assembly:
+//
+// * networking/multiple_networks/primary_networks/about-user-defined-networks.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="opening-default-network-ports-udn_{context}"]
+= Opening default network ports on user-defined network pods
+
+By default, pods on a user-defined network are isolated from the default network. This means that default network pods, such as those running monitoring services (Prometheus or Alertmanager) or the {product-title} image registry, cannot initiate connections to UDN pods.
+
+To allow default network pods to connect to a user-defined network pod, you can use the `k8s.ovn.org/open-default-ports annotation`. This annotation opens specific ports on the UDN pod for access from the default network.
```

Comment:
```suggestion
To allow default network pods to connect to a user-defined network pod, you can use the `k8s.ovn.org/open-default-ports` annotation. This annotation opens specific ports on the UDN pod for access from the default network.
```

Reply:
Fixed. Thanks!

Reply:
@zhaozhanqi whenever you ack this one, I'll get it merged. Thanks!

Reply:
/lgtm